arangod/Graph/Enumerators/WeightedTwoSidedEnumerator.h

```diff
@@ -101,9 +101,29 @@ class WeightedTwoSidedEnumerator {
 
     [[nodiscard]] bool isEmpty() const { return _queue.empty(); }
 
-    [[nodiscard]] std::vector<CalculatedCandidate> getQueue() const& {
-      return _queue;
-    };
+    [[nodiscard]] std::vector<Step*> getLeftLooseEnds() {
```

Comment:
nit: This method can be made const.

Reply:
afaict, it can't, because then the `Step *` want to become `const` and hilarity ensues. I am on the case getting this code straightened out; it's just taking a bit. In particular I need to sort out lifetimes of these steps....